server/data/series.js

```diff
@@ -1,5 +1,17 @@
 import {List} from 'immutable';
 
+export function getPositionInSeries(tags: {}): ?number {
+  const chapterString = 'chapter';
+  const chapterTag = Object.keys(tags).find((tag) => tag.toLowerCase().startsWith(chapterString));
```

Comment:
There's not support in IE10 - are we sure that `babel` is giving us this?

I don't think we get it from [es2015](https://github.com/wellcometrust/wellcomecollection.org/blob/master/client/.babelrc).